Service/CallService.php

```diff
@@ -42,9 +56,28 @@ public function call(
             else {
                 $response = $this->client->requestAsync($method, $url, $config);
             }
-        } catch (\GuzzleHttp\Exception\GuzzleException $e) {
+        } catch (ServerException|ClientException $e) {
+            $log->setResponseStatus('bla');
```

Comment:
? questionable ?

Reply:
Is this just for testing purposes?
Can't you use actual data here?

Reply:
Somehow the GuzzleException does not always contain responses, the status is something I'm still looking for, will replace it by '' for now.

Reply:
Check, maybe a todo comment is in order here?
Or check if the GuzzleException does have a response and set it if it does?